src/main/java/frc/robot/competition/Robot.java

```diff
@@ -77,10 +77,14 @@ public void robotPeriodic() {
   }
 
   @Override
-  public void disabledInit() {}
+  public void disabledInit() {
+    m_robotContainer.m_feedback.disabledColorPattern();
```

Comment:
Instead use `RobotModeTriggers.disabled().onTrue(m_feedback.disabledColorPattern().runsWhenDisabled())`
in `CompetitionRobotContainer`

Reply:
fixed